core-accounting-fe/src/core-accounting/resource/table.tsx

```diff
@@ -1,17 +1,30 @@
 import React from 'react'
 
-import { List, Datagrid, TextField, NumberField, useRecordContext} from 'react-admin'
+import { List, Datagrid, TextField, NumberField, useRecordContext, ReferenceField, DateField, SearchInput, SavedQueriesList, FilterLiveSearch, FilterList, FilterListItem } from 'react-admin'
 
 import { MoneyField } from './MoneyField'
 import { TimeAgoField } from './TimeAgoField'
 
-export const Table: React.FC<{datagridProps?: any; tableProps?: any, children?: any}> = ({datagridProps, tableProps, children}) => {
+import { PolymorphicReferenceField } from './PolymorphicReferenceField'
+import { Card, CardContent } from '@mui/material'
+import MailIcon from '@mui/icons-material/MailOutline';
+
+export const PostFilterSidebar = () => (
```

Comment:
again, this is application specific. We can add the ability for filter / aside / sort params to be passed into the table component (similar to what we are doing with the children), but they shouldn't live within the core-accounting-fe pkg

Reply:
please remove